node-rapids.code-workspace

```diff
@@ -229,7 +229,12 @@
             "**/.cmake-js/**": true,
             "**/node_modules/**": true
         },
-        "typescript.tsdk": "node-rapids/node_modules/typescript/lib"
+        "typescript.tsdk": "node-rapids/node_modules/typescript/lib",
+        "workbench.colorCustomizations": {
+            "activityBar.background": "#193409",
+            "titleBar.activeBackground": "#23480C",
+            "titleBar.activeForeground": "#F5FCF0"
+        }
```

Comment:
These should go in your VSCode user settings.

```suggestion
```

Reply:
Yes, sorry, those went into the workspace by default for some reason and I accidentally added it but will revert.